pkg/apis/flowcontrol/validation/validation.go

```diff
@@ -83,13 +83,18 @@ func ValidateFlowSchema(fs *flowcontrol.FlowSchema) field.ErrorList {
 	allErrs := apivalidation.ValidateObjectMeta(&fs.ObjectMeta, false, ValidateFlowSchemaName, field.NewPath("metadata"))
 	specPath := field.NewPath("spec")
 	allErrs = append(allErrs, ValidateFlowSchemaSpec(fs.Name, &fs.Spec, specPath)...)
-	if mand, ok := internalbootstrap.MandatoryFlowSchemas[fs.Name]; ok {
+	if mand, ok := internalbootstrap.GetMandatoryFlowSchemasMap(true)[fs.Name]; ok {
+		// The old config objects are a subset of the new config objects,
+		// as far as identify is concerned. The specs may differ.
+		mandOld := internalbootstrap.GetMandatoryFlowSchemasMap(false)[fs.Name]
```

Comment:
if we assume as @wojtek-t does (and I think it's a good point), if we have more than two sets of flowschemas to consider, the `false` probably needs to be string, right?

Reply:
I was thinking that the arguments to that function will be the values of the relevant Features. As we add and remove relevant Features, the function's signature will change and the compiler will help us find all the call sites, so we know that we correctly updated them.

Reply:
Lately I have updated the PR to use a FeatureGate instead of the collection of relevant Feature values.